docs/ROADMAP.md

```diff
@@ -0,0 +1,442 @@
+    * ie. test set and performance metric
+
+__Output:__ Function (sequence of tuples) mapping number of flipped labels to
+performance metric values
+
+__Applications:__
+
+- Measure of robustness to mislabeled data
+- Measure of overfitting
+- Influence measure for points or clusters in the training data
+
+__Type__: Model performance metric
+
+
+Stability of methodology
+------------------------
+
+While models are generally selected to maximize some measure of performance,
+the final choice of model also carries an inherent dependence on the
+methodology used to train it.
+For example, if a different train-test split were used, the final model would
+likely be slightly different.
+These analyses measure the effects of methodological choices on model
+performance, with the goal of minimizing them.
+Note that, for any of these approaches which uses resampling to estimate
+variability, computation cost needs to be taken into account as a part of the
+design.
+
+__Application scope:__ These generally require a model specification and
+training set, and can be applied post-hoc (to assess error in reported
+results) or prior to training (to help select methodology, using an assumed
+prior model choice).
+
+### Train-test splitting
+
+When training a model, a test set is typically held out at the beginning so as
+to provide an unbiased estimate of model performance on unseen data.
+However, the size of this test set itself influences the quality of this
+estimate.
+To assess this, we consider the variability and bias in performance metrics as
+the test set size varies.
+This is estimated by splitting the input dataset at different proportions and
+training and testing a model across these.
+
+__Input:__
+
+- Dataset
+    * ie. training set
+- Model specification
+- Performance metric
+
+__Output:__ Function (sequence of tuples) mapping train-test split size to
+performance estimates represented as a mean with confidence bounds
+
+__Applications:__
+
+- Measure of error (variance/bias) in test set performance evaluations
+- Selection of train-test split size to minimize bias and variance
+- Deviation from average-case performance
+
+__Type__: Model performance confidence metric
+
+### Cross-validation folds
+
+Similarly, the choice of validation methodology will influence the quality of
+estimates obtained using it.
+PRESC assesses this by computing model cross-validation (CV) model performance
+estimates across different numbers of folds.
+
+__Input:__
+
+- Dataset
+    * ie. training set
+- Model specification
+- Performance metric
+
+__Output:__ Function (sequence of tuples) mapping number of CV folds to
+performance estimates represented as a mean with confidence bounds
+
+__Applications:__
+
+- Measure of error (variance/bias) in CV performance evaluations
+- Selection of number of CV folds to minimize bias and variance
+- Deviation from average-case performance
+
+__Type__: Model performance confidence metric
+
+
+Other
+-----
+
+### Novelty
```

Comment:
Structurally, I  think the contents of the entire Novelty section can be integrated under other themes. 
the comparison as a function of time is not fundamentally different that a comparison between multiple models trained over different subsets of available data contemporaneously available. 

I really like the clarity of the themes wihtout the need for a catch-all "other" category. Perhaps we can either roll these contents into their appropriate sections or just exclude this section for now?